mtp_api/apps/transaction/api/cashbook/urls.py

```diff
@@ -5,6 +5,7 @@
 
 urlpatterns = [
     url(r'^transactions/$', csrf_exempt(views.TransactionList.as_view()), name='transaction-list'),
+    url(r'^transactions/locked/$', views.LockedTransactionList.as_view(), name='transaction-locked'),
```

Comment:
Might've been nice to see this as `transactions/?locked=true`, and just another filter on the overall transactions list.